docs/HowToGuides/GettingStarted.md

```diff
@@ -368,6 +351,82 @@ select the following schemes:
   run.
 <!-- TODO: Insert SourceKit/stdlib specific instructions? -->
 
+### Using Ninja with Xcode
+
+Although it's possible to build the toolchain entirely with Xcode via `--xcode`,
+a more efficient and robust option is to integrate a Ninja build with Xcode.
+This is also convenient in that you can navigate, build, run, edit and debug in
```

Comment:
Nit: missing serial comma before “and debug”

Reply:
A serial comma in this non-ambiguous case is regarded a preference as far as I know. Not to say I have a strong preference—will do.